### PR TITLE
Planner 1593 deep cloning shadow variables map docs

### PR DESCRIPTION
**Pull request by ge0ffrey**

### CODE REVIEW

File: optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/domain/clone/deepcloning/TestdataDeepCloningEntity.java [hunk 1]
```diff
@@ -68,6 +72,8 @@ public void setValue(TestdataValue value) {
     }
 
     @DeepPlanningClone
+    @CustomShadowVariable(sources = {@PlanningVariableReference(variableName = "value")},
```
Comment (triceo) [inline optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/domain/clone/deepcloning/TestdataDeepCloningEntity.java]: Why this update to `TestData`, when there is no update to any of the actual tests?

File: optaplanner-docs/src/main/asciidoc/LocalSearch/LocalSearch-chapter.adoc [hunk 2]
```diff
@@ -283,15 +283,14 @@ Advanced configuration:
 [[tabuSearchAlgorithm]]
 === Algorithm Description
 
-Tabu Search works like Hill Climbing, but it maintains a tabu list to avoid getting stuck in local optima.
+Tabu Search is a Local Search that maintains a tabu list to avoid getting stuck in local optima.
```
Comment (triceo) [inline optaplanner-docs/src/main/asciidoc/LocalSearch/LocalSearch-chapter.adoc]: This change is not related to PLANNER-1593.

File: optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/domain/clone/deepcloning/TestdataDeepCloningEntity.java (hunk 1 above)
Comment (ge0ffrey) [reply optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/domain/clone/deepcloning/TestdataDeepCloningEntity.java]: the field is called shadowVariable - but it wasn't one already. Just makes more sense like this.

File: optaplanner-docs/src/main/asciidoc/LocalSearch/LocalSearch-chapter.adoc (hunk 2 above)
Comment (ge0ffrey) [reply optaplanner-docs/src/main/asciidoc/LocalSearch/LocalSearch-chapter.adoc]: Indeed, request from JustinG that I am lumping in this already small PR. For small PR's I think it's fine to combine such things (it's a separate commit for history of course).

**Comment by ge0ffrey** [comment]
Issue renamed, good point. Thanks
